RUNNING_ON_YOUR_DESKTOP.md

```diff
@@ -0,0 +1,63 @@
+# Ubuntu Core Kiosk
+
+**Disclaimer:** *This isn't about how to deploy on "Classic" linux systems (i.e. not "Ubuntu Core") such as Ubuntu Desktop or Ubuntu Server. For these systems see the `daemon` configuration option in [REFERENCE.md](REFERENCE.md)*
```

Comment:
> **Disclaimer:** *This isn't about how to deploy on "Classic" linux systems (i.e. not "Ubuntu Core")

One too many levels of negatives for me. "this is how to deploy to Ubuntu Core, not classic systems such as Ubuntu Desktop.." sounds better.

Or maybe you you meant this is for classic systems? It seems to me like you are talking about classic and the daemon is for core.